services/conda/conda.tester.js

```diff
@@ -87,27 +86,13 @@ t.create('downloads (skip prefix, relabel)')
 
 t.create('unknown package')
   .get('/d/conda-forge/some-bogus-package-that-never-exists.json')
-  .expectJSON({ name: 'conda|downloads', value: 'invalid' })
+  .expectJSON({ name: 'conda', value: 'not found' })
 
 t.create('unknown channel')
   .get('/d/some-bogus-channel-that-never-exists/zlib.json')
-  .expectJSON({ name: 'conda|downloads', value: 'invalid' })
+  .expectJSON({ name: 'conda', value: 'not found' })
 
 t.create('unknown info')
```

Comment:
This test seems unnecessary. This was enforced by the regex / pattern, and still is.